scanner/src/main/kotlin/scanners/fossid/FossIdConfig.kt

```diff
@@ -111,4 +122,23 @@ internal data class FossIdConfig(
             )
         }
     }
+
+    /**
+     * Create a [FossIdNamingProvider] helper object based on the configuration stored in this object.
+     */
+    fun createNamingProvider(): FossIdNamingProvider {
+        val namingProjectPattern = options[NAMING_PROJECT_PATTERN_PROPERTY]?.apply {
```

Comment:
Using `also` here (and below) instead of `apply` should get rid of the log-related test failures.

Reply:
I still have a slight preference for using `also` instead of `apply` to avoid the need for the explicit `FossIdConfig.` prefix, but I'll approve anyway.